search/src/App.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import {
+	ReactiveBase, CategorySearch, SingleRange, ResultCard, ReactiveList
+} from '@appbaseio/reactivesearch';
+import logo from './logo.svg';
+import './App.css';
+
+class App extends Component {
+	render() {
+		return (
+				<ReactiveBase
+				app="kibana_sample_data_ecommerce"
```

Comment:
@hlydecker should this not reference the weedid dataset? Was this branch out-of-date with your demo when we merged it?